specification/src/main/asciidoc/jakarta-concurrency.adoc

```diff
@@ -1,7 +1,7 @@
 :sectnums:
-= Jakarta Concurrency Specification, Version 3.0
+= Jakarta Concurrency Specification, Version 3.1
 
-Copyright (c) 2013, 2023 Oracle and/or its affiliates. All rights reserved.
+Copyright (c) 2013, 2024 Oracle and/or its affiliates. All rights reserved.
```

Comment:
Should we wait to update the copyright until we have changes to the spec that happen in 2024?

Reply:
Yes, thank you for spotting that.  It's a mistake, and I added https://github.com/jakartaee/concurrency/pull/348/commits/e743973a54e2a5f1354e9592824f398026b67c6d to correct it.